### PR TITLE
added an introduction

### PR DESCRIPTION
**Pull request by dhelle2**
intro/docs and text for introduction

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ function HomepageHeader() {
             className="button button--secondary button--lg"
             to="/docs/introduction"
           >
-            Documentation ✏
+            Introduction/Documentation ✏
```
Comment (clee231) [inline src/pages/index.tsx:24]: Maybe make this a shorter name? Maybe "About"?

File: docusaurus.config.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ module.exports = {
           title: 'Docs',
           items: [
             {
-              label: 'Docs',
+              label: 'Intro/Docs',
```
Comment (clee231) [inline docusaurus.config.js:44]: Let's keep this as `Docs`.

File: docusaurus.config.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ module.exports = {
           type: 'doc',
           position: 'left',
           docId: 'introduction',
-          label: 'Docs',
+          label: 'Intro/Docs',
```
Comment (clee231) [inline docusaurus.config.js:27]: Let's keep this as `Docs`.

File: docs/introduction.md [hunk 4]
```diff
@@ -4,4 +4,13 @@ sidebar_position: 1
...
+
+ACM@UIC functions as a bridge between the CS department and its students. We host and participate in events which help keep the UIC techie community close, including LAN parties, lectures by guest speakers, company presentations,
+activities with other tech related clubs and an office to hang out in and talk or study adjacent to the cs lounge
+on the second floor of SELE on campus. Our discord and slack servers also provide virtual communities online. Addtionally we have a number of special interest groups (SIGS) which meet regularly to discuss and work on specific topics
+and projects.
+
+ACM also provides computing resources such as servers, software and a working network in our office, accounts for
+members along with the opportunity to obtain practice maintaining and improving these resources through
+activities and SIGS. See the following pages for more information and documentation.`
```
Comment (clee231) [inline docs/introduction.md:16]: Let's make this an "About" page instead of putting this in documentation.

**Review by clee231 — CHANGES_REQUESTED** [review]

**Review by clee231 — APPROVED** [review]
LGTM!
Luna Guily Tetracids Mesatipelvic

**Review by bmiddha — APPROVED** [review]

**Pull request merged**